code/src/Components/Style.css

```diff
@@ -0,0 +1,202 @@
+* {
+	margin: 0;
+	padding: 0;
+	box-sizing: border-box;
+}
+
+h1 {
+	font-size: 20px;
+	margin-bottom: 30px;
+	font-weight: 400;
+}
+
+h2 {
+	grid-column: 1/4;
+	font-weight: 200;
+	font-size: 18px;
+}
+
+main {
+	width: 90%;
+	max-width: 500px;
+	margin: 30px auto;
+	font-family: menlo;
+	display: grid;
```

Comment:
Remember to add `grid-template-column` and `grid-template-rows` when declaring the `display grid` 🥰

Reply:
Thanks for the input, I thought then when I use `display grid` that the default value is as many rows needed depending on how the code is structured and column 1fr. When I add `grid-templet-column` 1fr and `grid-template-rows` auto nothing is changed in the code. I will need to look up grid a bit more 😃